tasks/main.yml

```diff
@@ -0,0 +1,64 @@
+---
+
+###
+### Main entrypoint
+###
+
+- name: Check if Helm binary exists
+  shell: command -v helm >/dev/null 2>&1
+  register: helm_exist
```

Comment:
AFAIK, we're using underscore (`_`) to prefix vars registered during the execution of the role, to "namespace" them. right @cytopia ?

Reply:
Nice feedback! Thx! Fixed!